calrissian/main.py

```diff
@@ -50,6 +50,9 @@ def add_arguments(parser):
     parser.add_argument('--tool-logs-basepath', type=Text, nargs='?', help='Base path for saving the tool logs')
     parser.add_argument('--conf', help='Defines the default values for the CLI arguments', action='append')
 
+    parser.add_argument('--pod-priority-class', type=Text, nargs='?', help='Priority Class Name assigned to the pod')
```

Comment:
+1 on the pragmatic approach. I wonder tough if at some point the growing number of arguments to the CLI will not become cumbersome. Maybe at some point having a "pod template" acting as a JSON PATCH to add all the k8s config specifics could be a solution.